.travis.yml

```diff
@@ -22,4 +22,4 @@ before_script:
     - npm install
 
 script:
-    - grunt
+    - grunt lint
```

Comment:
`grunt test` is a better task to be calling from Travis I think, because you can add any kind of test to it and not restrict it to just linting right now